example_workspace/src/image_transport_sample_app/jni/build.gradle

```diff
@@ -6,7 +6,7 @@ android {
     defaultConfig {
         applicationId = 'com.example.image_transport_sample'
         minSdkVersion 24
-        targetSdkVersion 24
+        targetSdkVersion 26
```

Comment:
Just out of curiosity: Is this update required because one of the other changes or is it actually unrelated? In `config.sh` we set `android-24` by default, although that is more related to `minSdkVersion` afaik.

Reply:
Release build failed in lintVitalRelease task, saying that minimum acceptable target sdk was 26.
It's not really necessary, as I also later changed install script to always use a debug build with the sample apps in order to avoid apk signing.

Reply:
Google Play Store is not accepting apps with `targetSdkVersion` < 26 since November, 2018. While this is not an issue for test apps, it's a good idea to start using 26 at least.